swiss/src/app/landsraad/landsraad.component.ts

```diff
@@ -43,7 +43,12 @@ export class LandsraadComponent implements OnInit {
     this.questionPaths = this.db.list("landsraad/questions").snapshotChanges().pipe(
       map(
         snapshots => {
-          return snapshots.map(snapshot => "landsraad/questions/" + snapshot.key)
+          let snapshotsFiltered = snapshots.filter(snap => {
+            const question = snap.payload.val()
+            // as we now have to fill-in DB object even if no decret is selected, filter out "no selected" decret
+            return question["decretType"] !== "- Žádný -"
```

Comment:
I would rather filter on empty value

Reply:
Hmm but empty value might be confusing for the user. Let's imagine this situation:
1. user selects decret
2. user changes opinion because (s)he agrees with allies house that that house will submit the decret
3. user wants to remove the decret -> empty value in the list is not that visible than "- Žádný -" (it's more explicit)

Also before the users select some decret it's better in my opinion to have there "- Žádný -" (to be explicit) than just empty value. This is the same as for actions where before selecting action, there is "- Žádná -" in the dropdown.

Reply:
Hm I think the issue here is that you are saving the user string to the DB instead of some enum value, which is then mapped to a string.  Like it's done for actions (types, visibilities etc.). So the function is the same, it's more about cleaner approach to code. But this is less a priority, we can merge it this way if you don't have much time, but add a TODO that this should be refactored.

Reply:
Ok will do. And will try to check the actions (I briefly recall the value - string approach).

Reply:
Ok done. In the `landsraad.component.ts`, there is a same filtering `snapshot.payload.val()["questionType"] !== ""` in two places. But that will be improved in the other PR, where I have common function `questionEligibleForDisplay`.